<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySerial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidgetMode">
      <widget class="QWidget" name="pageExpert">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="hLayFunc">
          <item>
           <widget class="QToolButton" name="btnSerial">
            <property name="text">
             <string>Serial</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnData">
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnPlot">
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnCorrection">
            <property name="text">
             <string>Correction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnSimulate">
            <property name="text">
             <string>Simulate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnHistory">
            <property name="text">
             <string>History</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnData">
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnLog">
            <property name="text">
             <string>Log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSetting">
            <property name="text">
             <string>Setting</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QStackedWidget" name="stackedWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageEasy">
       <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,4,6" columnstretch="3,7">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="hLayOperation">
          <item>
           <widget class="QToolButton" name="tBtnSwitch">
            <property name="text">
             <string>start/stop</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnPause">
            <property name="text">
             <string>pause</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnZoom">
            <property name="text">
             <string>zoom</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnCrop">
            <property name="text">
             <string>crop</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnPeak">
            <property name="text">
             <string>peak</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnFWHM">
            <property name="text">
             <string>FWHM</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnImg">
            <property name="text">
             <string>img</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_Operation">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gLayInfo">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelIntegrationTime">
              <property name="text">
               <string>Integration time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_ms">
              <property name="text">
               <string>ms</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="1" rowspan="2">
         <layout class="QGridLayout" name="gLayPlot"/>
        </item>
        <item row="2" column="0">
         <widget class="QTableView" name="tableViewValue"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="action_en"/>
    <addaction name="action_zh_simple"/>
    <addaction name="action_zh_traditional"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionLite"/>
    <addaction name="actionDark"/>
    <addaction name="actionOSX_Lite"/>
    <addaction name="actionOSX_Dark"/>
    <addaction name="actionHelloKitty"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionEasy"/>
    <addaction name="actionExpert"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuTheme"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_en">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>en</string>
   </property>
  </action>
  <action name="action_zh_simple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zh_simple</string>
   </property>
  </action>
  <action name="action_zh_traditional">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zh_traditional</string>
   </property>
  </action>
  <action name="actionLite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lite</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionOSX_Lite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OSX_Lite</string>
   </property>
  </action>
  <action name="actionOSX_Dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OSX_Dark</string>
   </property>
  </action>
  <action name="actionHelloKitty">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HelloKitty</string>
   </property>
  </action>
  <action name="actionEasy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionExpert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Expert</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btnSerial</tabstop>
  <tabstop>btnData</tabstop>
  <tabstop>btnPlot</tabstop>
  <tabstop>btnLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
